### DISCUSSION TITLE
inspircd does not start up - says no <bind> block when I have created one.

### DISCUSSION THREAD

**Discussion by SomewhatAwake** [inspircd/inspircd#2060] [Support]
This command
ubuntu@irc-server:/usr/sbin$ ./inspircd --config /inspircd/inspircd.conf 
Produces this.
InspIRCd - Internet Relay Chat Daemon
See /INFO for contributors & authors

InspIRCd Process ID: 4957
Could not read "/inspircd/inspircd.conf" for include
Possible configuration error: you have not defined any bind blocks.
You will need to do this if you want clients to be able to connect!

However, in the config file at /inspircd/inspircd.conf:
"
bind address="the ip i want to bind to"
      port="6667"
      type="clients"
      free="no"
"
Zero clue what's going on, tried running as root, tried adding some redundant variables, still do not know what's going on

**Comment by sadiepowell** [comment]
>Could not read "/inspircd/inspircd.conf" for include

This means either that file doesn't exist or isn't readable by the user you're running as. If you're still getting the same error as root then that file probably does not exist. Are you sure that's where you've stored your config?

How have you installed InspIRCd? Those paths look like they're from our Docker container. If so the configs are stored at `/inspircd/conf` not `/inspircd`.

**Reply by SomewhatAwake** [reply]
Hi there,

I originally installed from apt, but it couldnt access /etc/inspircd, so I manually moved the config files to /inspircd. I've double checked the path I'm using, and the config file is where I've specified.

**Reply by sadiepowell** [reply]
Ah, Debian's package is currently broken because their AppArmor profile is misconfigured as a result of them passing the wrong paths to the build system. We recommend you install our official .deb package from https://github.com/inspircd/inspircd/releases/tag/v3.16.1 instead which works a lot better.

To do this you should run `sudo apt purge inspircd`, download the .deb for your system, then do `sudo apt install ./<FILENAME>.deb`

**Reply by SomewhatAwake** [reply]
Thanks a lot. Will build from that.
